pkg/plugins/platform/templates.go

```diff
@@ -69,10 +69,8 @@ services:
     image: {{ .router_image }}:{{ .router_tag }}
     container_name: nginx-proxy
     ports:
-      - "80:80"
-      - {{ .mailhogport }}:{{ .mailhogport }}
-      - {{ .dbaport }}:{{ .dbaport }}
-      - {{ .dbport }}:{{ .dbport }}
+      {{ range $port := .ports }}- "{{ $port }}:{{ $port }}"
```

Comment:
So this is actually a mistake that I introduced without realizing it, and have known would need to be fixed when we get here 😭. The format of the ports directive is "host:container". We only want the host port to change, not the internal container ports. The ports variable probably needs to be a map which maps external -> internal ports.